soccer/gameplay/plays/Attack_Goal/distraction_attack.py

```diff
@@ -257,3 +265,20 @@ def on_enter_shoot(self):
             priority=10)
         self.add_subbehavior(
             skills.pivot_kick.PivotKick(), 'shooting', required=True)
+
+
+
+    @classmethod
+    def score(cls):
```

Comment:
I've just added situational code to distraction, although it probably doesn't really need a score function for right now as the score function isn't really doing anything since there is no offset.